defaults/main.yml

```diff
@@ -59,7 +59,7 @@ nim_waku_rln_relay_content_topic: '/toy-chat/2/luzhou/proto'
 nim_waku_rln_relay_dynamic: true
 nim_waku_rln_relay_eth_account_address: '0000000000000000000000000000000000000000'
 nim_waku_rln_relay_eth_contract_address: '0x4252105670fE33D2947e8EaD304969849E64f2a6'
-#nim_waku_rln_relay_eth_client_address: 'http://geth.example.org:8545'
```

Comment:
Why did you remove this line?

Reply:
Because it's commented out? should I leave it in?

Reply:
It's intentionally commented out to make this `mandatory` check trigger if it's not defined:
https://github.com/status-im/infra-role-nim-waku/blob/eed1a10ccea90c3b33af845ba54334530467a625/templates/docker-compose.yml.j2#L66
And the comment is there for documentation purposes with an example value.

Reply:
Addressed in [8044c33](https://github.com/status-im/infra-role-nim-waku/pull/11/commits/8044c33ffb92b3ee73cba677a090330ff638b70c), moved `nim_waku_rln_relay_tree_path` above the config vars for the dynamic mode, like in the template of the docker-compose file.